synfig-studio/src/synfigapp/actions/layerbrush.h

```diff
@@ -0,0 +1,131 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file layerbrush.h
+**	\brief Template File
+**
+**	\legal
+**	......... ... 2014 Ivan Mahonin
+**
+**	This file is part of Synfig.
+**
+**	Synfig is free software: you can redistribute it and/or modify
+**	it under the terms of the GNU General Public License as published by
+**	the Free Software Foundation, either version 2 of the License, or
+**	(at your option) any later version.
+**
+**	Synfig is distributed in the hope that it will be useful,
+**	but WITHOUT ANY WARRANTY; without even the implied warranty of
+**	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+**	GNU General Public License for more details.
+**
+**	You should have received a copy of the GNU General Public License
+**	along with Synfig.  If not, see <https://www.gnu.org/licenses/>.
+**	\endlegal
+*/
+/* ========================================================================= */
+
+/* === S T A R T =========================================================== */
+
+#ifndef __SYNFIG_APP_ACTION_LAYERBRUSH_H
+#define __SYNFIG_APP_ACTION_LAYERBRUSH_H
```

Comment:
remove the initial double underscore `__`. We are avoiding using it in new files, as it is reserved in C/C++

References:
https://stackoverflow.com/questions/43461615/c-double-underscore-inside-of-define-macro
https://devblogs.microsoft.com/oldnewthing/20230109-00/?p=107685
https://github.com/protobuf-c/protobuf-c/issues/188